### DISCUSSION TITLE
MQTT Issues

### DISCUSSION THREAD

**Discussion by matthew-hz** [tagyoureit/nodejs-poolController#438] [Q&A] [answered]
I recently pulled njsPC and am having issues sending MQTT commands from Home Assistant and other tools. I had njsPC installed on another device with the last pull in February, so I checked and MQTT worked fine. I wanted to see if anyone else has experienced issues or had to make changes to their configuration before I spend more time troubleshooting.

**Comment by rstrouse** [comment, marked as answer]
Pull njsPC some extensions were ignoring wildcard topics.

**Reply by matthew-hz** [reply]
Everything looks good so far now!

**Reply by vj950** [reply]
I can also confirm that mqtt is working fine now.
